build fails on win for 'err.h'

https://github.com/columbia-irt/rtptools/pull/97 is a big step forward of rtptools getting better.

In the transition, It fails to build on Win.
1>d:\documents\dev\rtptools\rtptools-1.23\utils.c(34): fatal error C1083: Cannot open include file: 'err.h': No such file or directory
1>utils.obj : error LNK2019: unresolved external symbol _warnx referenced in function _hpt

Even it fails to build on Win at the moment, moving to err() is great way to go.

Also it has the error message.
1>utils.obj : error LNK2019: unresolved external symbol _inet_aton referenced in function _host2ip
Windows do not have inet_aton for some unknown reason, but has inet_pton().
https://stackoverflow.com/questions/2420663/windows-equivalent-of-inet-aton
https://msdn.microsoft.com/en-us/library/windows/desktop/cc805844(v=vs.85).aspx

I hope somebody from MS is watching all the discussion this github. :)

--- a/sysdep.h
+++ b/sysdep.h
@@ -33,13 +33,13 @@
 
 #if defined(WIN32) || defined(__WIN32__)
 
-#include <winsock2.h>  /* For NT socket */
-#include <ws2tcpip.h>  /* IP_ADD_MEMBERSHIP */
+#include <winsock2.h>
+#include <ws2tcpip.h>
 #include <windows.h>
-#include <stdio.h>     /* stderr */
-#include <time.h>      /* time_t */
+#include <stdio.h>
+#include <time.h>
 
-#define HAVE_STDINT_H 1
+#define HAVE_ERR 0
 #define RTP_BIG_ENDIAN 0
 
 /* Determine if the C(++) compiler requires complete function prototype  */

--- a/utils.c
+++ b/utils.c
@@ -31,13 +31,13 @@
 #include <sys/types.h>
 #include <stdlib.h>
 #include <string.h>
-#include <err.h>
 
 #ifndef WIN32
 #include <sys/socket.h>
 #include <arpa/inet.h>
 #include <netinet/in.h>
 #include <netdb.h>
+#include <err.h>
 #endif
 
 #include "sysdep.h"

--- a/win/rtpdump.vcxproj
+++ b/win/rtpdump.vcxproj
@@ -87,6 +87,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="../utils.c" />
+    <ClCompile Include="../compat-err.c" />
     <ClCompile Include="../multimer.c" />
     <ClCompile Include="../notify.c" />
     <ClCompile Include="../rd.c" />

--- a/win/rtpplay.vcxproj
+++ b/win/rtpplay.vcxproj
@@ -89,6 +89,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="../utils.c" />
+    <ClCompile Include="../compat-err.c" />
     <ClCompile Include="../compat-hsearch.c" />
     <ClCompile Include="../multimer.c" />
     <ClCompile Include="../notify.c" />

--- a/win/rtpsend.vcxproj
+++ b/win/rtpsend.vcxproj
@@ -89,6 +89,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="../utils.c" />
+    <ClCompile Include="../compat-err.c" />
     <ClCompile Include="../multimer.c" />
     <ClCompile Include="../notify.c" />
     <ClCompile Include="../rtpsend.c" />

--- a/win/rtptrans.vcxproj
+++ b/win/rtptrans.vcxproj
@@ -89,6 +89,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="../utils.c" />
+    <ClCompile Include="../compat-err.c" />
     <ClCompile Include="../multimer.c" />
     <ClCompile Include="../notify.c" />
     <ClCompile Include="../rtptrans.c" />

